dev_guide/getting_traffic_into_cluster.adoc

```diff
@@ -133,10 +146,212 @@ xref:../admin_guide/high_availability.adoc#admin-guide-high-availability[VIP] ad
 - In a non-cloud environment, by configuring ingressIP range (`ingressIPNetworkCIDR`),
 `service.type=LoadBalancer`, and `service.port.ingressIP`.
 
-The administrator must ensure the external IPs are routed to the nodes and local firewall rules on all nodes allow access to the open port.
+The administrator must ensure the external IPs are routed to the nodes and local
+firewall rules on all nodes allow access to the open port.
+
+.Example External IP Configured within a Service Definition
+----
+$ oc create -f - <<INGRESS
+apiVersion: v1
+kind: Service
+metadata:
+  name: postgresql-ingress
+spec:
+  externalIPs:
+  - 10.9.54.100 <1>
+  ports:
+  - port: 5432
+    protocol: TCP
+  selector:
+    name: postgresql
+  type: LoadBalancer
+INGRESS
+----
+<1> The IP address assigned to the external IP.
+
+The external IP address is not managed by the underlying Kubernetes
+infrastructure and must be maintained and provided by a cluster administrator.
+While external IPs provide a solution for accessing services on the {product-tlte}
+cluster, there are several shortcomings:
+
+* The cluster administrator user must ensure the external IP is not in any range
+that the cluster can use. The user needs to work with the administrator that
+controls the network external to the cluster to assign the external IP address,
+a DNS name for the IP, and ensure traffic to the IP can get to the nodes in the
```

Comment:
The DNS name is not strictly necessary... and in fact may not be assigned by the operator of the cluster.

For instance, if I use an externalIP on a public cluster (e.g. openshift.com), I need to configure the DNS I control for benbennett.com to point to the IP address that was assigned.